dotnet/samples/KernelSyntaxExamples/Example79_ApiManifest.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Plugins.MsGraph.Connectors.CredentialManagers;
+using Microsoft.SemanticKernel.Plugins.OpenApi;
+using Microsoft.SemanticKernel.Plugins.OpenApi.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+namespace Examples;
+
+// This example shows how to use the ApiManifest based plugins
+public class Example79_ApiManifest : BaseTest
+{
+    private void WriteSampleHeadingToConsole(string pluginToTest, string functionToTest, KernelArguments? arguments, params string[] pluginsToLoad)
```

Comment:
nit: move all private members to the bottom of the class to emphasize the actual example.